src/map.h

```diff
@@ -4,8 +4,19 @@
 #include <xcb/xcb.h>
 #include "win.h"
 
-void map_init();
+typedef enum
+{
+	LABEL_KEY_MODE_AVY,
+	LABEL_KEY_MODE_ALPHA,
+
+	LABEL_KEY_MODE_DEFAULT = LABEL_KEY_MODE_AVY,
+	
+	LABEL_KEY_MODE_MAX,
```

Comment:
do we really need the max mode here?

Reply:
You're right that it's unnecessary. I just did it by habit for when defining an enumeration of possible types so validity can be checked.  I'll drop it.